README.md

```diff
@@ -1,568 +1,181 @@
+This makes incredibly difficult to iterate on and deploy a production-grade generative AI application. Here are common issues today:
 
-### Why should I use `aiconfig` instead of things like [configurator](https://pypi.org/project/configurator/)?
+- increased complexity to iterate on prompts and model parameters
+- dificulty in switching between models across providers
+- limited collaboration with prompt engineers, who might be domain experts and not developers
+- limited evaluation capabilities for prompt and model performance
 
-It helps to have a [standardized format](http://aiconfig.lastmileai.dev/docs/overview/ai-config-format) specifically for storing generative AI prompts, inference results, model parameters and arbitrary metadata, as opposed to a general-purpose configuration schema.
+AIConfig helps unwind complexity by separating prompts, model parameters, and model-specific logic from your application.
 
-With that standardization, you just need a layer that knows how to serialize/deserialize from that format into whatever the inference endpoints require.
+- simplifies application code -- simply call `config.run()`.
+- open the `aiconfig` in a playground to iterate quickly. prompt engineers can start here and handoff the `aiconfig` to developers.
+- version control and evaluate the `aiconfig` - it's the AI artifact for your application.
 
-### This looks similar to `ipynb` for Jupyter notebooks
+<div style="text-align:center;">
+    <img src="aiconfig-docs/static/img/aiconfig_dataflow.png" alt="AIConfig flow" style="width:40%;">
+</div>
 
-We believe that notebooks are a perfect iteration environment for generative AI -- they are flexible, multi-modal, and collaborative.
+## Get involved
 
-The multi-modality and flexibility offered by notebooks and [`ipynb`](https://ipython.org/ipython-doc/3/notebook/nbformat.html) offers a good interaction model for generative AI. The `aiconfig` file format is extensible like `ipynb`, and AI Workbook editor allows rapid iteration in a notebook-like IDE.
+We are rapidly developing AIConfig! We welcome PR contributors and ideas for how to improve the project.
 
-_AI Workbooks are to AIConfig what Jupyter notebooks are to `ipynb`_
+- [Join the conversation on Discord](https://discord.com/invite/xBhNKTetGx) - `#aiconfig` channel
+- [Open an issue for feature requests](https://github.com/lastmile-ai/aiconfig/issues)
+- [Read our contributing guide](https://aiconfig.lastmileai.dev/docs/contributing/)
 
-There are 2 areas where we are going beyond what notebooks offer:
+## Release Cadence
 
-1. `aiconfig` is more **source-control friendly** than `ipynb`. `ipynb` stores binary data (images, etc.) by encoding it in the file, while `aiconfig` recommends using file URI references instead.
-2. `aiconfig` can be imported and **connected to application code** using the AIConfig SDK.
+We currently release new tagged versions of the `pypi` and `npm` packages on Mondays. Hotfixes go out at any time during the week.
```

Comment:
Can you add a link to the Changeling or add a Changelogs section?

Reply:
Should we just say releases 'each week' instead of on Mondays, and hotfixes go out when completed? We should still aim for Mondays but we shouldn't rush stuff out EOD Monday just to follow this 'contract'. Better to release Tuesday after ensuring everything is good in such cases

Reply:
done